### ISSUE TITLE
question about the training loss

### ISSUE THREAD

**Issue by cherise215** [Jason-Qiu/Transfer_Knowledge_from_Language_to_Electrocardiography#2]
In your paper, on page 2, you said " In addition to the traditional cross-entropy loss. "  Isn't that the cross-entropy loss includes the classification labels during computation? I don't really get it. Sorry for re-posting. But I don't think issue 1 is really closed.

_Originally posted by @cherise215 in https://github.com/Jason-Qiu/Transfer_Knowledge_from_Language_to_Electrocardiography/issues/1#issuecomment-1557734177_

**Comment by wan363** [comment]
Have you run your code? I think your code is shit! Your code have just done n-points window filtering to ECG signal during preprocess，some running results in two "ipynb " files's is wrong，you use labels in training phase but you still call it zero-shot，your code just use 13 time domain features and 9 frequency features. "utils.py " import "load" from "evaluate.py" but "evaluate.py" don't have "load". Do you think it makes sense to import "utils.py" in "translate.py", import "load" from "evaluate.py" in "utils. py", and read "translations_864.csv" in "evaluate. py".
